mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -46,7 +46,7 @@ var (
 				Type: schema.TypeBool,
 				Optional: true,
 				Computed: true,
-				Description: `Whether master is accessbile via Google Compute Engine Public IP addresses.`,
+				Description: `Whether master is accessible via Google Compute Engine Public IP addresses.`,
```

Comment:
```suggestion
				Description: `Whether Kubernetes master is accessible via Google Compute Engine Public IPs.`,
```

This field has drifted from its website page, which also includes "Kubernetes" as a prefix to master and shortened IP addresses to IPs. WDYT?

Reply:
Excellent, thanks!